step.yml

```diff
@@ -99,40 +99,18 @@ inputs:
       description: |-
         The track in which you want to assign the uploaded app. 
         
-        Can be one of the built-in tracks: internal, alpha, beta, rollout, production.
+        Can be one of the built-in tracks: internal, alpha, beta, production.
         
         Or you can set your custom track name as well. 
         
         For example: `pre-release`, or any of your closed track you added in Google Play Developer Console.
       is_required: true
-  - user_fraction: "0.5"
+  - user_fraction:
     opts:
       title: User Fraction
       description: |-
-        __Only available if `track` is `rollout`.__
-
-        Portion of the users who should get the staged rollout version of the app. Accepts values between 0.0 and 1.0.
-      is_required: true
-      value_options:
-        - "0.05"
-        - "0.1"
-        - "0.15"
-        - "0.2"
-        - "0.25"
-        - "0.3"
-        - "0.35"
-        - "0.4"
-        - "0.45"
-        - "0.5"
-        - "0.55"
-        - "0.6"
-        - "0.65"
-        - "0.7"
-        - "0.75"
-        - "0.8"
-        - "0.85"
-        - "0.9"
-        - "0.95"
+        Portion of the users who should get the staged version of the app. Accepts values between 0.0 and 1.0.
```

Comment:
Some users would think that they can enter 1 (`between` could be understood like that), add an example how to set it to "100%"

Reply:
Updated description.